lib/membrane/rtp/session_bin.ex

```diff
@@ -136,12 +139,12 @@ defmodule Membrane.RTP.SessionBin do
         """
       ],
       extensions: [
-        spec: [:vad],
+        spec: [extension_t()],
         default: [],
         description: """
         List of extensions. Currently `:vad` is only supported.
         * `:vad` will turn on Voice Activity Detection mechanism firing appropriate notifications when needed.
-        Should be set only for audio tracks. For more information refer to `Membrane.RTP.VAD` module documentation.
+           Should be set only for audio tracks. For more information refer to `Membrane.RTP.VAD` module documentation.
```

Comment:
Can we check if this doesn't introduce one more tab? By running `mix docs`.

Reply:
It seems that it does not change anything but I reverted it back.